roles/crashfiles/README.md

```diff
@@ -0,0 +1,72 @@
+# network.healthchecks.crashfiles
+
+## Overview
+The `network.healthchecks.crashfiles` role allows monitoring of crash files on network devices. This helps detect system crashes and stability issues, ensuring device reliability. The role provides a comprehensive health check view that shows the status of crash files and overall system health.
+
+## Features
+- Monitor presence of crash files
+- Track crash file details
+- Generate alerts for system crashes
+- Provide detailed health check status (PASS/FAIL)
+- Show crash file statistics and information
+
+## Variables
+| Variable Name   | Default Value | Required | Type  | Description                                      |
+|----------------|--------------|----------|-------|--------------------------------------------------|
+| `details` | false | no | bool | Whether to include detailed crash file information in output |
+
+## Usage
+
+### Example: Monitoring Crash Files
+```yaml
+- name: Monitor crash files
+  ansible.builtin.include_role:
+    name: network.healthchecks.crashfiles
+  vars:
+    ansible_network_os: cisco.ios.ios
+    details: true
+    ignore_errors: false
+  register: crash_result
+
+- name: Display crash files health check results
+  ansible.builtin.debug:
+    var: crash_result.health_checks
+```
+
+### Output: Crash Files Health Check Status
+```json
+{
+    "health_checks": {
+        "crash_files": {
+            "check_status": "PASS",
+            "total_crash_files": 0
+        },
+        "crash_files_summary": {
+            "total_crash_files": 0,
+            "crash_files": []
+        },
+        "status": "PASS"
```

Comment:
like cpu related healthcheck , we also need name key here

Reply:
name key as in?